src/js/pouchPersisted.js

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+/* global fluid, floe, PouchDB */
+
+(function ($, fluid) {
+
+    "use strict";
+
+    // Base grade for persistence of model components to Pouch
+    // Implementing grades should ensure that what's integral to
+    // reconstructing the particular instance is stored in the
+    // model
+    fluid.defaults("floe.dashboard.pouchPersisted", {
+        gradeNames: ["floe.dashboard.eventInTimeAware"],
```

Comment:
This looks like a straightforward refactor to align with similar use cases. I'll look at the prior work you identify & let you know if I have any questions.